content/en/docs/tasks/access-application-cluster/ingress-minikube.md

```diff
@@ -135,23 +135,21 @@ The following file is an Ingress resource that sends traffic to your Service via
 
 1. Create `example-ingress.yaml` from the following file:
 
-    ```yaml
-    apiVersion: networking.k8s.io/v1beta1
-    kind: Ingress
-    metadata:
-      name: example-ingress
-      annotations:
-        nginx.ingress.kubernetes.io/rewrite-target: /
-    spec:
-     rules:
-     - host: hello-world.info
-       http:
-         paths:
-         - path: /*
-           backend:
-             serviceName: web
-             servicePort: 8080
-    ```
+        apiVersion: networking.k8s.io/v1beta1
+        kind: Ingress
```

Comment:
Hi, @pilosus. Would it be possible to leave the `yaml` type for code output.
Cleaning up the example string (4 spaces from item number) should work:
1. Create `example-ingress.yaml` from the following file:

```yaml
        apiVersion: extensions/v1beta1
        kind: Ingress
        metadata:
          name: example-ingress
          annotations:
            nginx.ingress.kubernetes.io/rewrite-target: /
        spec:
         rules:
         - host: hello-world.info
           http:
             paths:
             - path: /*
               backend:
                 serviceName: web
                 servicePort: 8080
    ```

Reply:
Hi, @kbhawkey! If you take a look at the current version of the file in [master](https://github.com/kubernetes/website/blob/master/content/en/docs/tasks/access-application-cluster/ingress-minikube.md) branch you'll see that GitHub renders it just fine. It seems to be a ``Hugo`` problem specifically, not ``Markdown``'s. People are discussing it [here](https://github.com/gohugoio/hugo/issues/4171)
I've tried some other ways to fix it, but with no luck.
I know that it really looks like an ugly hack, but at least it works. As far as I see syntax highlighting is not affected in anyway. So may be it's okay to use a non-fenced code block here even if it's not consistent internally across the docs?

Reply:
Hi, @pilosus. If you indent four spaces to start the code type directive, and then indent the YAML past the directive, then the output renders correctly -- or at least it did for me.

Reply:
@kbhawkey oh I see what you mean. This works indeed. But the code inside highlighted block gets indented more than in the blocks below:

https://imgur.com/a/CIuTVvH

I my PR code in all the blocks are aligned:
https://imgur.com/a/p0IWDg7

I don't know if it's important or not. I actually don't mind to add language YAML directive if this over-ident is okay

Reply:
Right. The code sample is in an indented block and is indented in the list. I think that is OK.
Will let others chime in.